cana/boolean_node.py

```diff
@@ -636,6 +636,43 @@ def bias(self):
 		"""
 		return sum(self.outputs) / 2**self.k
 
-
-
-
+	def c_sensitivity(self, c, mode="default", max_k=0):
+		"""give the c-sensitivity of this node's Boolean function
```

Comment:
@xuan-w , can you modify the description to use Google documentation style so sphinx will understand what are Args and what are Returns, etc? See other functions for examples.